rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake

```diff
@@ -230,16 +169,10 @@ ament_target_dependencies(
   "rosidl_typesupport_opensplice_c")
 foreach(_pkg_name ${rosidl_generate_interfaces_DEPENDENCY_PACKAGE_NAMES})
   set(_msg_include_dir "${${_pkg_name}_DIR}/../../../include/${_pkg_name}/msg/dds_opensplice_c")
-  set(_srv_include_dir "${${_pkg_name}_DIR}/../../../include/${_pkg_name}/srv/dds_opensplice_c")
```

Comment:
An `.idl` file can include messages, services and actions. The include directory for services and actions isn't necessary because `idlpp` doesn't generate any headers for services and actions - only `.cpp` files. Only for messages `idlpp` generates header files.